packages/nhost-js/src/clients/nhost.ts

```diff
@@ -1,10 +1,41 @@
 import { HasuraAuthClient } from '@nhost/hasura-auth-js'
 import { HasuraStorageClient } from '@nhost/hasura-storage-js'
 
-import { NhostFunctionsClient } from '../clients/functions'
-import { NhostGraphqlClient } from '../clients/graphql'
-import { urlFromParams } from '../utils/helpers'
 import { NhostClientConstructorParams } from '../utils/types'
+
+import { createAuthClient } from './auth'
+import { createFunctionsClient, NhostFunctionsClient } from './functions'
+import { createGraphqlClient, NhostGraphqlClient } from './graphql'
+import { createStorageClient } from './storage'
+
+export function createNhostClient({
+  refreshIntervalTime,
+  clientStorageGetter,
+  clientStorageSetter,
+  clientStorage,
+  clientStorageType,
+  autoRefreshToken,
+  autoSignIn,
+  adminSecret,
+  devTools,
+  start = true,
+  ...urlParams
+}: NhostClientConstructorParams): NhostClient {
+  return new NhostClient({
+    refreshIntervalTime,
+    clientStorageGetter,
+    clientStorageSetter,
+    clientStorage,
+    clientStorageType,
+    autoRefreshToken,
+    autoSignIn,
+    adminSecret,
+    devTools,
+    start,
+    ...urlParams
+  })
+}
+
```

Comment:
This function is not used anymore except in tests, and they could run against `new NhostClient(params)`

If you however want to keep this, you can simplify:
```suggestion
export const createNhostClient = (params: NhostClientConstructorParams) => new NhostClient(params)
```